packages/material-ui/src/RadioGroup/RadioGroup.d.ts

```diff
@@ -5,7 +5,7 @@ import { FormGroupProps, FormGroupClassKey } from '../FormGroup';
 export interface RadioGroupProps
   extends StandardProps<FormGroupProps, RadioGroupClassKey, 'onChange'> {
   name?: string;
-  onChange?: (event: React.ChangeEvent<{}>, value: string) => void;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>, value: string) => void;
```

Comment:
I guess this is fine. It's not technically correct since this only applies if users actually use `Radio` components as children but it's probably the best compromise between run-time and compile-time.

Reply:
Agree it may not be not 100% precise, but the typing would be okay even if the children were `Switch`es or `Checkbox`es, so it could be a reasonable default.

Reply:
I agree. Although we don't do any runtime checks we document it as a wrapper for `Radio` only and rely on the onChange implementation of that component.

@oliviertassinari Should we add a runtime check when mapping the children or do you think the documentation is enough?